persistent-mysql/persistent-mysql.cabal

```
@@ -1,5 +1,5 @@
 name:            persistent-mysql
-version:         2.13.1.5
+version:         2.13.2
```

No API changes, so we can do a patch: 
```suggestion
version:         2.13.1.6
```